docfiles/pxtweb/cookieCompliance.ts

```diff
@@ -126,6 +138,10 @@ namespace pxt {
         const loadAI = (window as any).loadAppInsights;
         if (loadAI) {
             loadAI(includeCookie)
+            analyticsLoaded = true;
+            while (pendingCallbacks.length) {
+                pendingCallbacks.pop()();
```

Comment:
are you supposed to call these callbacks?

Reply:
I am calling them. Maybe I should add a comment since I had to stare at that line for a bit to figure that out...

Reply:
Please. I'm really confuse about what's happening here.

Reply:
There are two sets of parentheses, one for `pop` and one to call the callback. I'll add a comment.

Reply:
what happens when the client overrides pxtError / pxtEvent?

Reply:
Not sure what you mean by that